samples/word/02-paragraphs/insert-in-different-locations.yaml

```diff
@@ -1,6 +1,7 @@
-id: word-basics-insert-in-different-locations
-name: Insert locations.
-description: Shows how to insert content using the different available locations.
+order: 2
+id: word-paragraphs-insert-in-different-locations
+name: Insert at different locations
```

Comment:
Suggest changing `name` attribute value to:  **Insert content at different locations**